src/event.rs

```diff
@@ -51,6 +51,8 @@ pub enum Event {
         previous_name: Option<XorName>,
         /// Age of the node
         age: u8,
+        /// Indication that is has been relocated during startup.
+        startup_relocation: bool,
```

Comment:
Out of curiosity, but why do we need this?

Reply:
To differentiate between natural relocation and startup phase induced relocation.

Reply:
I get that, but why do we need to differentiate it?